code/game/gamemodes/miniantags/guardian/types/protector.dm

```diff
@@ -8,15 +8,41 @@
 	tech_fluff_string = "Boot sequence complete. Protector modules loaded. Holoparasite swarm online."
 	bio_fluff_string = "Your scarab swarm finishes mutating and stirs to life, ready to defend you."
 	var/toggle = FALSE
+	/// The shields the guardian has, and brings with it as it moves
+	var/list/connected_shields = list()
 
 /mob/living/simple_animal/hostile/guardian/protector/ex_act(severity)
 	if(severity == 1)
-		adjustBruteLoss(400) //if in protector mode, will do 20 damage and not actually necessarily kill the summoner
+		adjustBruteLoss(200) //if in protector mode, will do 20 damage and not actually necessarily kill the summoner
 	else
 		..()
 	if(toggle)
 		visible_message("<span class='danger'>The explosion glances off [src]'s energy shielding!</span>")
 
+
+/mob/living/simple_animal/hostile/guardian/protector/Manifest()
+	. = ..()
+	if(toggle && cooldown > world.time)
```

Comment:
So they'll need to reenter their host, toggle and then manifest for it to become active. Is this desired behaviour or should it also activate on toggle when manifested?

Reply:
Every other guardian has to enter body to toggle, so I kept it consistant